src/fidesops/ops/migrations/versions/55d61eb8ed12_add_default_policies.py

```diff
@@ -48,6 +45,7 @@
 FIDESOPS_AUTOGENERATED_STORAGE_KEY = "fidesops_autogenerated_storage_destination"
 AUTOGENERATED_ACCESS_KEY = "download"
 AUTOGENERATED_ERASURE_KEY = "delete"
+STRING_REWRITE_STRATEGY_NAME = "string_rewrite"
```

Comment:
is it OK to edit this file by hand? i hesitated before doing so since it's a generated migration file...

Reply:
This is fine since it's functionally equivalent, we're just cleaning up the constants.

Reply:
oh, nice - thanks for clarifying that @pattisdr!